test/cpp/end2end/tls_private_key_offload_end2end_test.cc

```diff
@@ -114,7 +115,7 @@ class TlsPrivateKeyOffloadTest : public ::testing::Test {
   std::unique_ptr<Server> server_ = nullptr;
   std::thread* server_thread_ = nullptr;
   std::string server_addr_;
-  std::shared_ptr<grpc_core::CustomPrivateKeySigner> signer_;
+  std::shared_ptr<grpc_core::PrivateKeySigner> signer_;
```

Comment:
No need for the `grpc_core::` prefix here.  This file is testing the C++ public API, so it should use the types there, not the C-core types.

Same thing throughout this file, anywhere that we're referring to the `PrivateKeySigner` type.